megamek/i18n/megamek/client/messages.properties

```diff
@@ -976,6 +976,7 @@ CommonMenuBar.GameMenu=Game
 CommonMenuBar.helpSkinning=Skinning How-to
 CommonMenuBar.Inferno=Inferno({0})
 CommonMenuBar.helpAbout=About MegaMek
+CommonMenuBar.helpResetNags=Reset Readme Settings
```

Comment:
Not really sure that this text accurately describes what the functionality does. Maybe "Reset Nag Settings" instead?

Reply:
I had no idea what to call it, even less so for German. OK let's try Nag :)